### PR TITLE
Ability to import fossa package files

### LINKED ISSUE #728: Add support for fossa input to workspace page
similar to how you can drag and drop a package-lock.json file, this issue is for dragging and dropping a fossa analyze output onto the workspace.

the schema looks like:

```json
 {
      "Name": "package",
      "Type": "commonjspackage",
      "Manifest": "package",
      "Build": {
        "Artifact": "default",
        "Context": null,
        "Succeeded": true,
        "Imports": [
          "npm+path-is-absolute$",
          "npm+fs.realpath$",
          "npm+inflight$",
          "npm+inherits$",
          "npm+minimatch$",
          "npm+once$"
        ],
        "Dependencies": [
          { "locator": "npm+setprototypeof$1.1.0" },
        {
          "locator": "npm+handlebars$4.1.0",
          "imports": ["npm+async$2.6.2", "npm+source-map$0.6.1", "npm+optimist$0.6.1", "npm+uglify-js$3.4.9"]
        },
        { "locator": "npm+no-case$2.3.2", "imports": ["npm+lower-case$1.1.4"] },
        { "locator": "npm+clean-css$4.2.1", "imports": ["npm+source-map$0.6.1"] },
        { "locator": "npm+component-emitter$1.2.1" },
        {
          "locator": "npm+readdirp$2.1.0",
          "imports": [
            "npm+set-immediate-shim$1.0.1",
            "npm+graceful-fs$4.1.11",
            "npm+minimatch$3.0.4",
            "npm+readable-stream$2.0.6"
          ]
        },
        ]
      }
    }
  ]
```

An initial implementation could just pull all the 'locator' properties and build them into coordinates.

`<type>+<name>$<revision>`
or
`<type>+<namespace>/<name>$<revision>`

### PR DESCRIPTION
**Pull request by storrisi**
Resolves #728

### CODE REVIEW

File: src/components/UserManagedList.js [hunk 1]
```diff
@@ -168,6 +173,21 @@ export default class UserManagedList extends SystemManagedList {
     return { coordinates }
   }
 
+  getListFromFossaPackage(dependencies) {
+    const coordinates = []
+    for (const dependency in dependencies) {
+      const locator = dependencies[dependency].locator.split(/[\s+$/]+/)
+      coordinates.push({
+        type: locator[0],
+        provider: 'npmjs',
```
Comment (geneh) [inline src/components/UserManagedList.js]: Should other package managers be supported besides NPM?

Comment (storrisi) [reply src/components/UserManagedList.js]: Honestly, i'm not so experienced about Fossa , so i don't know which kind of package managers could be supported.
If you take a look to the code, we are actually forcing the same thing for the `package-lock` file, so i guess is an expected behavior?

Comment (storrisi) [reply src/components/UserManagedList.js]: @geneh We could actually improve this code, and match the field `type` with the related provider (ex. `npm -> npmjs`) . 
We can extend this improvement also for the `package-lock` file parsing.

Thoughts?

Comment (geneh) [reply src/components/UserManagedList.js]: Good! Isn't package-lock relevant only to NPM? Fossa seems to support multiple package managers: https://github.com/fossas/fossa-cli/blob/master/README.md#Supported-Environments

**Pull request merged**